samples/StatusCodePagesSample/hosting.json

```diff
@@ -1,4 +0,0 @@
-{
-  "server": "Microsoft.AspNetCore.Server.Kestrel",
-  "server.urls": "http://localhost:5001/base"
```

Comment:
This one was using a different port. Does that matter for any tests or anything? @Tratcher do you remember if this was deliberate?